manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: focus@will.focusatwill
+PackageVersion: 1.6.0.44

```

This must match what is shown in control panel
```suggestion
PackageVersion: 1.6.0
```